java/src/main/java/com/google/oak/crypto/hpke/Context.java

```diff
@@ -20,47 +20,143 @@
 
 // TODO(#3642): Implement Java Hybrid Encryption.
 public final class Context {
-  public static final class SenderRequestContext {
+  public static final class SenderRequestContext implements AutoCloseable {
+    private final long nativePtr;
+    public SenderRequestContext(long nativePtr) {
+      this.nativePtr = nativePtr;
+    }
+
+    static {
```

Comment:
Since theyre all separate public static classes, i believe they need their own static blocks for the System.loadLibrary since any of them could technically be called first and would need to load the library.

Added try-catch for all the System.loadLibrary calls.